tc-health-client/README.md

```diff
@@ -171,11 +171,11 @@ Sample configuarion file:
 
 # Files
 
-* /etc/trafficcontrol-cache-config/tm-health-client.json
-* /etc/logrotate.d/tm-health-client-logrotate
-* /usr/bin/tm-health-client
-* /usr/lib/systemd/system/tm-health-client.service
-* /var/log/trafficcontrol-cache-config/tm-health-client.json
+* /etc/trafficcontrol-health-client/tc-health-client.json
+* /etc/logrotate.d/tc-health-client-logrotate
+* /usr/bin/tc-health-client
+* /usr/lib/systemd/system/tc-health-client.service
+* /var/log/trafficcontrol-health-client/tc-health-client.json
```

Comment:
@zrhoffman interesting that you should bring this up.  @rob05c insists the directory names are the same as the RPM name/project folder.  I personally prefer that /etc/trafficcontrol and /var/log/trafficcontrol are used but relented on that argument.  It would be nice if all TC log files are under /var/log/trafficcontrol

Reply:
>  @rob05c insists the directory names are the same as the RPM name/project folder
In package names, the component separator is `-`, and in paths, the component separator is `/`. So, for a package named `trafficcontrol-health-client`, the equivalent path *is* `trafficcontrol/health-client`. To script that (which seems like an overoptimization already), it's true that you'd need to convert from kebab case to a path, but we already convert camelCase to snake_case:

https://github.com/apache/trafficcontrol/blob/08ec4ad715282ec3763b111eae70c99491d0f89a/lib/go-util/join.go#L73-L75

Representing the same information differently in different places is nothing new, even within the scope of ATC alone.

Reply:
IMHO we should just use the project name, **trafficcontrol** for config and log directories, why overcomplicate it and proliferate directories and file names all over the filesystem.  I'm for using **/var/log/trafficcontrol/** and **/etc/trafficcontrol**

Reply:
@zrhoffman Rob, @rob05c, is okay with this.